draft-ietf-mls-protocol.md

```diff
@@ -1032,25 +1032,27 @@ represents a blank node and unmerged leaves are indicated in square
 brackets:
 
 ~~~ ascii-art
-       ...
-       /
-      _
-      |
-    .-+-.
-   /     \
-  _       Z[C]
- / \     / \
-A   _   C   D
+               ...
+               /
+              _
```

Comment:
Shouldn't this node be set to `Z[C]`? I think D will set it as "the top node"

Reply:
Ah, yes, as it is now.  I need to have someone outside the subtree remove G in order to clear it.  I'll update the description.